### ISSUE TITLE
Make it easier to create a new turtle

### ISSUE THREAD

**Issue by JensLincke** [timfel/etoys-spur#3]
it is not clear how to create a new turtle...

the easiest was to paint directly on the world the turtle was moved...

**Comment by JensLincke** [comment]
Does one paint new turtles or does one paint patches?

**Comment by timfel** [comment]
You paint turtles. Patches are the "shared memory".

**Comment by timfel** [comment]
Patches have a value for each pixel. The value determines how "transparent" the patch is (higher value = less transparent). The patch can "diffuse" (meaning its value spreads out over the neighbouring pixels over time) and they can "evaporate" (meaning their value goes down over time).

**Comment by timfel** [comment]
patches cannot move, they have one value per pixel, and you cannot easily "move" that value.
turtles can move and they can also carry variables, so to "send" data around with direction, you need to use turtles. to just "spread information" you can use patches.

**Comment by JensLincke** [comment]
So, the painted turtles should have the position you painted them...

**Comment by timfel** [comment]
yes, but there is a bug there, they move to the top left, if there is nothing in the topleft, because the painting is cropped before it is applied to kedama, i think.

**Comment by timfel** [comment]
i will make you a video

**Comment by timfel** [comment]
here is a video
[kedama-draw.zip](https://github.com/timfel/etoys-spur/files/426899/kedama-draw.zip)

**Comment by JensLincke** [comment]
but when painting without an existing turtle... the turtles are all moved 

https://www.dropbox.com/s/mlmn7q8mbzfwa4a/Squeak%21%20%28C__Users_Jens_Dropbox_bin_squeak_Etoys6.0beta-7605-32bit-r201606301459-Windows_Jens_Kedama_Baum_01.image%29%2019.08.2016%2014_03_05.mp4?dl=0
